catalyst-gateway/tests/Earthfile

```diff
@@ -53,10 +53,18 @@ test-lint-openapi:
 test-builder:
     FROM gateway+builder-src
 
+    WORKDIR $HOME/build
```

Comment:
I think you dont need this line, the `FROM gateway+builder-src` should put you already in the build folder